appveyor.yml

```diff
@@ -0,0 +1,64 @@
+build: false
+shallow_clone: true
+platform: x86
+clone_folder: c:\projects\phpspec
+
+environment:
+    matrix:
+        - PHP_DOWNLOAD_FILE: php-5.3.3-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.3.29-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.4.44-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.5.29-nts-Win32-VC11-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.6.14-nts-Win32-VC11-x86.zip
+        - PHP_DOWNLOAD_FILE: php-7.0-nts-windows-vc14-x86-r2e38918.zip
+
+matrix:
+    allow_failures:
+        - PHP_DOWNLOAD_FILE: php-5.3.3-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.3.29-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.4.44-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.5.29-nts-Win32-VC11-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.6.14-nts-Win32-VC11-x86.zip
+        - PHP_DOWNLOAD_FILE: php-7.0-nts-windows-vc14-x86-r2e38918.zip
+
+skip_commits:
+    message: /\[ci skip\]/
+
+cache:
+    - c:\php -> appveyor.yml
+    - '%LOCALAPPDATA%\Composer -> composer.json'
+    - .phpunit -> phpunit
+    - vendor
+
+init:
+    - SET PATH=c:\php;%PATH%
+    - SET COMPOSER_NO_INTERACTION=1
+    - SET PHP=1
+    - SET ANSICON=121x90 (121x90)
+
+install:
+    - IF EXIST c:\php (SET PHP=0) ELSE (mkdir c:\php)
+    - cd c:\php
+    - IF %PHP%==1 (IF %PHP_DOWNLOAD_FILE%==php-7.0-nts-windows-vc14-x86-r2e38918.zip (appveyor DownloadFile http://windows.php.net/downloads/snaps/php-7.0/r2e38918/%PHP_DOWNLOAD_FILE%) ELSE (appveyor DownloadFile http://windows.php.net/downloads/releases/archives/%PHP_DOWNLOAD_FILE%))
+    - IF %PHP%==1 7z x %PHP_DOWNLOAD_FILE% -y > 7z.log
+    - IF %PHP%==1 echo @php %%~dp0composer.phar %%* > composer.bat
+    - appveyor DownloadFile https://getcomposer.org/composer.phar
+    - copy php.ini-production php.ini /Y
+    - echo date.timezone="UTC" >> php.ini
+    - echo extension_dir=ext >> php.ini
+    - echo extension=php_openssl.dll >> php.ini
+    - echo extension=php_curl.dll >> php.ini
+    - echo extension=php_mbstring.dll >> php.ini
+    - echo extension=php_fileinfo.dll >> php.ini
+    - cd c:\projects\phpspec
+    - SET COMPOSER_ROOT_VERSION=dev-master
+    - composer update --prefer-source --no-progress --ansi
```

Comment:
installing from source forbids composer to use the cache (it caches downloaded archives, but it cannot cache ``git clone``). This makes the cache useless